internal/rbd/rbd_util.go

```diff
@@ -697,47 +697,67 @@ func (ri *rbdImage) trashRemoveImage(ctx context.Context) error {
 	return nil
 }
 
-func (ri *rbdImage) getCloneDepth(ctx context.Context) (uint, error) {
-	var depth uint
-	vol := rbdVolume{}
+// getCloneDepth walks the parents of the image and returns the number of
+// images in the chain. The `max` argument to the function is used to specify a
+// limit of the depth to check. When `max` depth is reached, no further
+// traversing of the depth is required.
+//
```

Comment:
Maybe we can remove this "//" from here.

Reply:
I've done that intentionally. An empty line between the short description of the function, and a details about the implementation.